test/Npgsql.Tests/ReaderTests.cs

```diff
@@ -706,6 +706,16 @@ public async Task SchemaOnly_next_result_beyond_end()
             Assert.False(reader.NextResult());
         }
 
+        [Test, IssueLink("https://github.com/npgsql/npgsql/issues/4124")]
+        public async Task SchemaOnly_GetDataTypeName_with_unsupported_type()
```

Comment:
Reader tests run 4 times (Default/Sequential + Multiplexing/NonMultiplexing). At some point we definitely should take a look at this and move some tests around.

Reply:
Sure thing